sentiment/app/main.py

```diff
@@ -3,6 +3,12 @@
 # from keras.models import load_model
 from transformers import AutoTokenizer, AutoModelForSequenceClassification, TextClassificationPipeline
 
+import argparse
+import os
+from ctypes import *
+
+kontain = CDLL("libkontain.so")
```

Comment:
Can we make this code tolerate absence of libkontain? Like disable related functionality if there is no library but keep working otherwise.